### PR TITLE
First stage html css

### PR DESCRIPTION
**Pull request by yousef-najeh**

### CODE REVIEW

File: first/style.css [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+    background-size: cover;
+    background-position: center;
+}
+
+.navBar {
+    position: fixed;
+    z-index: 3;
+    top: 0px;
+    display: flex;
+    flex-direction: row;
```
Comment (amonamoqadi) [inline first/style.css]: recheck this

File: first/style.css [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+}
+
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+.header{
+    height:600px;
```
Comment (amonamoqadi) [inline first/style.css]: ads space after :

File: first/style.css (hunk 1 above)
Comment (amonamoqadi) [reply first/style.css]: the default is row, why did you keep it? @yousefQJ19

Comment (yousef-najeh) [reply first/style.css]: i thought it was because the aligning of the elements

**Pull request merged**